src/Sylius/Behat/Page/Admin/Customer/IndexPage.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Customer;
+
+use Sylius\Behat\Page\Admin\Crud\IndexPage as BaseIndexPage;
+use Sylius\Behat\Page\Admin\Crud\IndexPageInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class IndexPage extends BaseIndexPage implements IndexPageInterface
```

Comment:
It would be cleaner if we would remove this class and instead of it declare `Sylius\Behat\Page\Admin\Crud\IndexPage` under `sylius.behat.page.admin.customer.index.class`